apps/guide/settings/base.py

```diff
@@ -267,6 +267,10 @@
     for version in WAGTAIL_GUIDE_VERSIONS
 ]
 
+LOCALE_PATHS = [
+    BASE_DIR / "locale",
+]
```

Comment:
Why do we have `custom_media/locale/`? Can't everything go in the global locale folder? Would be nice to have a single po per language.

Reply:
Looks like it's because `custom_media/locale/` already exists. I've deleted it so that all locale strings are generated to `BASE_DIR / "locale"` only.